### ISSUE TITLE
mixing of training/validation weights

### ISSUE THREAD

**Issue by wheatpennymaster** [BMEII-AI/RadImageNet#5]
The model weights are not re-instantiated before each call of run_model() in the folds loop. Therefore, the model has seen images from fold 0 (i.e. training set of fold 0) which are included in the validation set for fold 1. This results in an increase in validation performance after each fold is trained.

**Comment by wheatpennymaster** [comment]
Heads up, the training code for sarscovid2, pneuomina, hemorrhage, covid19 are also still bugged. Thanks for correcting the others.
